.polygon/rest.json

```diff
@@ -2786,7 +3063,7 @@
           {
             "name": "from",
             "in": "path",
-            "description": "The start of the aggregate time window.",
+            "description": "The start of the aggregate time window. Either a date with the format YYYY-MM-DD or a nanosecond timestamp.",
```

Comment:
i intentionally omitted this change in the go client since we expect millis but the python client handles time differently. should i update the descriptions in this case?

Reply:
I think it's probably fine since the python client handles all timestamps as ints, but I want to check with Zack just in case.

Reply:
never mind, the description already mentions all the possible options